core/domain/feedback_jobs_continuous.py

```diff
@@ -44,15 +46,31 @@ class FeedbackAnalyticsAggregator(jobs.BaseContinuousComputationManager):
 
     @classmethod
     def get_event_types_listened_to(cls):
+        """Get the event types that this class is subscribed to.
+
+        Returns:
+            list(str). List of event types that this class is subscribed to.
+        """
         return [feconf.EVENT_TYPE_NEW_THREAD_CREATED,
                 feconf.EVENT_TYPE_THREAD_STATUS_CHANGED]
 
     @classmethod
     def _get_realtime_datastore_class(cls):
+        """Get the realtime datastore class used by the realtime layer.
+
+        Returns:
+            class. Datastore class used by the realtime layer, which should
+            subclass BaseRealtimeDatastoreClassForContinuousComputations.
+        """
         return FeedbackAnalyticsRealtimeModel
 
     @classmethod
     def _get_batch_job_manager_class(cls):
+        """Get manager class.
+
+        Returns:
+            class. Manager class for continuous-running batch job.
```

Comment:
I think 'class' is a wrong type. This should be same as what is printed when you add `type(var)` in python. It should be 'FeedbackAnalyticsMRJobManager' I think. Please recheck!

Reply:
Done